tools/build/third_party/perfetto/perfetto_build_flags.h

```diff
@@ -32,5 +32,17 @@
 #define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_LINENOISE() (0)
 #define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_JSON() (0)
 #define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_PERCENTILE() (1)
+#define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_SQLITE() (1)
+#define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_METRICS() (1)
+#define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_FTRACE() (1)
+#define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_HTTPD() (1)
+#define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_SYSTEM_PROBES() (1)
+#define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_ANDROID_PROBES() (1)
+#define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_HEAP_GRAPHS() (1)
+#define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_GRAPHICS() (1)
+#define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_JSON_IMPORT() (0)
+#define PERFETTO_BUILDFLAG_DEFINE_PERFETTO_TP_FUCHSIA() (0)
```

Comment:
what breaks if this is on?

Reply:
We can turn on TP_FUCHSIA, I'll do that. TP_JSON_IMPORT fails the compilation though.

Reply:
Yes, GH is not making it clear that I was only asking about the Fuchsia one :)